src/values/acceleration-structure.cxx

```diff
@@ -0,0 +1,1484 @@
+/* © SPIRV-Interpreter @ https://github.com/mmoult/SPIRV-Interpreter
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+module;
+#include <algorithm>
+#include <array>
+#include <limits>
+#include <cmath>
+#include <memory>
+#include <stack>
+#include <string>
+#include <tuple>
+
+// TODO: plan to remove/change header(s) below
+#include <iostream>
+
+// TODO: probably want to reduce GLM files to link
+#define GLM_ENABLE_EXPERIMENTAL
+#include <glm/glm.hpp>
+#include <glm/ext.hpp>
+#include "../external/spirv.hpp"
+#include "type.hpp"
+#include "value.hpp"
+
+export module value.accelerationStructure;
+import value.aggregate;
+import value.primitive;
+
+namespace Util {
+    std::string repeatedString(const unsigned num, const std::string str) {
+        std::stringstream result("");
```

Comment:
You don't need to specify a blank string. Simply use:

std::stringstream result;